builds/any/rootfs/buster/common/overlay/etc/systemd/system/networking.service.d/switchdev-online.conf

```diff
@@ -0,0 +1,2 @@
+[Unit]
```

Comment:
Why are you adding this file twice at different locations? Only one copy should be enough.

Reply:
As I understand we need to have for overlay for stretch and a buster.